MovingBlocks/Terasology
# Apply special keyboards option in Input settings

### Contains

This PR adds new buttons on the Input settings screen to quickly change your key mappings if you have a special keyboard like AZERTY, DVORAK or NEO.
The mappings were already available through console commands but that would not save them in the settings and that was difficult to reach for a casual player.

There is a localization of the title and the buttons for both English and French.

Screenshot:
![screenshot from 2018-12-10 10-42-31](https://user-images.githubusercontent.com/12398255/49723915-594fcb00-fc68-11e8-9231-2f0239bcf126.png)


### How to test

1. On the main screen, open Settings then Input. You should see the new section "Special keyboards".
2. Try it by clicking on Apply for some special keyboard like AZERTY.
3. The changes in mapping should be instantly reflected in the binding buttons below (in Movement, Interaction, etc.) if they were supposed to change for the special keyboard you chose.
4. You might want to check in-game and ensure the new mappings are effectively applied.
* Note: the steps are also be available from the in-game Settings menu
